Dockerfile

```diff
@@ -1,38 +1,47 @@
-
-
-FROM ubuntu:21.04 as build
+FROM ubuntu:20.04 as build
 
 ENV LANG C.UTF-8
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN dpkg --add-architecture i386 && \
-    apt-get update && \
+RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y \
-        binutils:i386 \
-        gcc-10:i386 \
-        g++-10:i386 \
-        python3.10 \
-        python \
+        binutils \
+        gcc-10 \
+        g++-10 \
+	p7zip-full \
+        python3.9 \
         make \
         cmake \
+        curl \
         git \
         lld \
-        libsdl2-dev:i386 \
-        zlib1g-dev:i386 \
-        libbz2-dev:i386 \
-        libpng-dev:i386 \
-        libgles2-mesa-dev && \
-    ln -sf /usr/bin/python3.10 /usr/bin/python3 && \
-    ln -s /usr/bin/gcc-10 /usr/bin/gcc && \
-    ln -s /usr/bin/gcc-10 /usr/bin/cc && \
+        libsdl2-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        libpng-dev \
+        libgles2-mesa-dev && \    
     ln -s /usr/bin/g++-10 /usr/bin/g++ && \
-    ln -s /usr/bin/g++-10 /usr/bin/c++
+    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 10 && \
```

Comment:
in https://github.com/HarbourMasters/Shipwright/pull/567 i just had
```
    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 10 && \
    update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 10
```
what's the reasoning behind using the `ln`?

Reply:
Throwing errors that g++ was not installed. and using `ln` for gcc throwing errors because it WAS installed.